src/Contract/Operation/Unpackable.php

```diff
@@ -13,10 +13,7 @@
 
 /**
  * @template TKey
- * @template T of array{0: NewTKey, 1: NewT}
- *
- * @template NewTKey
- * @template NewT
+ * @template T
```

Comment:
I don't get this change. Was the existing annotation not working? Or is it not correct?

There's no static analysis check for `unpack` so this would be the time to add one if we're changing these

Reply:
Despite the fact that this was technically valid, I changed that because:

1. All operations interfaces are using 2 templates (`TKey` and `T`), `Unpackable` was using 4 (`TKey`, `T`, `NewTKey`, `NewT`)
2. Incoming templates (`TKey` and `T`) should be opaque, at least that's how it works on every other operations. We should not describe the structure of these or else it will never end. If we start to do that, where should we put the limit or depth of details?
3. By removing that, we can remove a bunch of "unsupported" lines from `phpstan-unsupported-baseline.neon`

Regarding the SA for `Unpack`, I will add it today for sure!

Reply:
Ok, I reverted the changes, I cannot get through it properly in PHPStan.